common/src/main/scala/org/apache/celeborn/common/RssConf.scala

```diff
@@ -1367,35 +1375,44 @@ object RssConf extends Logging {
       .withAlternative("rss.metrics.system.enabled")
       .categories("master", "worker", "metrics")
       .doc("When true, enable metrics system.")
+      .version("0.2.0")
       .booleanConf
       .createWithDefault(true)
 
   val METRICS_SAMPLE_RATE: ConfigEntry[Double] =
     buildConf("celeborn.metrics.sample.rate")
       .withAlternative("rss.metrics.system.sample.rate")
       .categories("master", "worker", "metrics")
+      .doc("It controls if Celeborn collect timer metrics for some operations. Its value should be in (0.0, 1.0).")
+      .version("0.2.0")
       .doubleConf
-      .checkValue(v => v >= 0.0 && v <= 1.0, "should be in [0.0, 1.0]")
+      .checkValue(v => v >= 0.0 && v <= 1.0, "should be in (0.0, 1.0)")
```

Comment:
Could we use a boolean to replace this config? Because seems it does the same thing as boolean and doesn't control any other logic.

Reply:
I think this config cannot be replaced by a boolean. It controls the sampling frequency. Check the logic below.

`  def needSample(): Boolean = {
    if (sampleRate >= 1) {
      true
    } else if (sampleRate <= 0) {
      false
    } else {
      Random.nextDouble() <= sampleRate
    }
  }`

Reply:
get it